tests/performance/log-results.js

```diff
@@ -20,42 +21,66 @@ const resultsFiles = [
 	},
 ];
 
-const performanceResults = resultsFiles.map( ( { file } ) =>
-	JSON.parse(
-		fs.readFileSync( path.join( __dirname, '/specs/' + file ), 'utf8' )
+// Results files recorded for the base commit.
+const baseResultsFiles = [
+	{
+		file: 'base-home-block-theme.test.results.json',
+		metricsPrefix: 'home-block-theme-',
+	},
+	{
+		file: 'base-home-classic-theme.test.results.json',
+		metricsPrefix: 'home-classic-theme-',
+	},
+];
+
+/**
+ * Parse test files into JSON objects.
+ *
+ * @param {Object[]} files
+ * @returns An array of parsed objects from each file.
+ */
+const parseResults = (files) => (
+	files.map( ( { file } ) =>
+		JSON.parse(
+			fs.readFileSync( path.join( __dirname, '/specs/' + file ), 'utf8' )
+		)
 	)
 );
 
 /**
- * Gets the array or metrics.
+ * Gets the array of metrics.
  *
- * @return {array} Metrics.
+ * @return {Object[]} Metrics.
  */
-const metrics = resultsFiles.reduce(
-	( result, { metricsPrefix }, index ) => {
-		return {
-			...result,
-			...Object.fromEntries(
-				Object.entries(
-					performanceResults[ index ] ?? {}
-				).map( ( [ key, value ] ) => [
-					metricsPrefix + key,
-					median(value),
-				] )
-			),
-		};
-	},
-	{}
-);
+const formatResults = (files) => {
+	const parsedResults = parseResults(files);
+
+	return files.reduce(
+		( result, { metricsPrefix }, index ) => {
+			return {
+				...result,
+				...Object.fromEntries(
+					Object.entries(
+						parsedResults[ index ] ?? {}
+					).map( ( [ key, value ] ) => [
+						metricsPrefix + key,
+						median(value),
+					] )
+				),
+			};
+		},
+		{}
+	);
+};
 
 const data = new TextEncoder().encode(
 	JSON.stringify( {
 		branch,
 		hash,
-		baseHash: '',
+		baseHash,
 		timestamp: parseInt( timestamp, 10 ),
-		metrics,
-		baseMetrics: [],
+		metrics: formatResults(resultsFiles),
+		baseMetrics: formatResults(baseResultsFiles),
```

Comment:
Missing spaces per WP coding standards. I see the CI workflows don't run here since it's a feature branch - would be great if we could find a way to set that up for the future.

Reply:
Thanks Felix, Fixed in https://github.com/10up/wordpress-develop/pull/36/commits/583342746bfd0176d3d32e036f22d54d5eb09622